test/vfd_swmr_dsetops_writer.c

```diff
@@ -0,0 +1,1771 @@
+/*
+ * Copyright by The HDF Group.
+ * Copyright by the Board of Trustees of the University of Illinois.
+ * All rights reserved.
+ *
+ * This file is part of HDF5.  The full HDF5 copyright notice, including
+ * terms governing use, modification, and redistribution, is contained in
+ * the COPYING file, which can be found at the root of the source code
+ * distribution tree, or in https://support.hdfgroup.org/ftp/HDF5/releases.
+ * If you do not have access to either file, you may request a copy from
+ * help@hdfgroup.org.
+ */
+
+/*
+ *  Purpose: To test writing operations for different dataset types.
+ *      Dataset types:
+ *      --dataset with compact layout
+ *      --dataset with contiguous layout
+ *      --datasets with chunked layout: single/implicit/fixed array/extensible array/btree2 indexes
+ *
+ *      Types of writes:
+ *      -- Sequential writes
+ *      -- Random writes
+ *      -- Regular hyperslab writes
+ *      -- Raw data modifications
+ */
+#include <err.h>
```

Comment:
These three headers may not be necessary.

Reply:
Fixed